index.html

```diff
@@ -189,6 +189,34 @@ <h4>Skills</h4>
             </ul>
         </section>
     </section>
+    <section class="contact">
+        <div class="contact-wrap">
+            <img src="images/contact_shapes.png" alt="" srcset="" class="contact-shapes-bg">
+            <h4 class="contact-title">Contact me</h4>
+            <p class="contact-desc">
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding.
+                I’d love to help with it!
+            </p>
+            <form action="https://formspree.io/f/xwkjrjye" method="post" class="contact-form">
+                <ul>
+                    <li>
+                        <input type="text" id="name" name="user_name" value="Lorem Ipsum" maxlength="30" />
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" value="loremipsum@gmail.com"  required />
+                    </li>
+                    <li>
+                        <textarea id="msg" name="user_message" maxlength="500">Write your message here...</textarea>
```

Comment:
- [ ] I was also able to submit the form while the message field is empty, kindly add `required` attribute to the `textarea` to prevent that.